frontend/packages/core/src/Input/radio.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import type { RadioProps as MuiRadioProps } from "@material-ui/core";
+import { Radio as MuiRadio } from "@material-ui/core";
+
+const StyledRadio = styled(MuiRadio)(
+  {
+    ".MuiIconButton-label": {
+      height: "24px",
+      width: "24px",
+      borderRadius: "100px",
+      boxSizing: "border-box",
+    },
+  },
+  props => ({
+    "&:hover > .MuiIconButton-label > div": {
+      border: props.checked ? "1px solid #283CD2" : "1px solid #2E45DC",
+    },
+  })
+);
+
+const Icon = styled.div(
+  {
+    height: "1.5rem",
```

Comment:
nit: since we're going with px, perhaps the few `rems` throughout should be converted to px.